src/main/java/com/uid2/admin/vertx/AdminVerticle.java

```diff
@@ -57,19 +59,20 @@ public void start(Promise<Void> startPromise) {
 
     private Router createRoutesSetup() {
         final Router router = Router.router(vertx);
-        router.route().handler(SessionHandler.create(LocalSessionStore.create(vertx)));
+        router.route().handler(SessionHandler.create(LocalSessionStore.create(vertx)).setSessionTimeout(32400000)); // 9 hr session timeout
```

Comment:
may be better to make it a config setting to have different timeout by environment ?